src/Libraries/PythonNodeModelsWpf/PythonNodeModelsWpf.csproj

```diff
@@ -39,10 +39,12 @@
       <HintPath>..\..\..\extern\avalonEdit\ICSharpCode.AvalonEdit.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="IronPython, Version=2.7.0.40, Culture=neutral, PublicKeyToken=7f709c5b713576e1, processorArchitecture=MSIL">
+    <Reference Include="IronPython">
+      <HintPath>..\..\..\extern\IronPython\IronPython.dll</HintPath>
```

Comment:
@phliberato why make this change, can you explain what the consequence is?

Reply:
The references of IronPython and Microsoft.Dynamic at PythonNodeModelsWpf project were with its path blank. I was having some troubles with IronPythonCompletionProvider, after of that I can run autocomplete at Python.